test/JDBC/input/BABEL-3201-vu-verify.mix

```diff
@@ -0,0 +1,247 @@
+
+INSERT INTO babel_3201_t_int VALUES (5);
+GO
+
+SELECT * FROM babel_3201_t_int;
+GO
+
+-- Remove all rows in table using DELETE TABLE, identity value inserted in next
+-- INSERT operation should be (new_reseed_value + increment).
+DELETE FROM babel_3201_t_bigint;
+GO
+
+DBCC CHECKIDENT(babel_3201_t_bigint, RESEED, 10);
+GO
+
+INSERT INTO babel_3201_t_bigint VALUES (5);
+GO
+
+SELECT * FROM babel_3201_t_bigint;
+GO
+
+-- Incorrect DBCC command option
+DBCC CHECKIDENT(babel_3201_t1) WITH NO_INFO;
+GO
+
+-- Invalid parameter 1
+DBCC CHECKIDENT(5);
+GO
+
+DBCC CHECKIDENT(babel_3201_t_int, RESEED, ) WITH NO_INFOMSGS;
+GO
+
+-- Invalid keyword
+DBCC CHECKIDENT(babel_3201_t1, RESEE);
+GO
+
+-- Invalid datatype
+DBCC CHECKIDENT(babel_3201_t1, RESEED, 1313abc);
+GO
+
+-- Unsupported DBCC command
+DBCC CHECKTABLE(babel_3201_t1);
+GO
+
+-- Invalid DBCC command
+DBCC FAKE_COMMAND(t1);
+GO
+
+-- Database undefined
+DBCC CHECKIDENT(fake_db.dbo.babel_3201_t1, NORESEED);
+GO
+
+-- Table undefined
+DBCC CHECKIDENT(babel_3201_t1, RESEED);
+GO
+
+-- Schema undefined
+DBCC CHECKIDENT(babel_3201_sch3.babel_3201_t1, NORESEED);
+GO
+
+-- Table does not have identity column
+DBCC CHECKIDENT(babel_3201_t2, RESEED);
+GO
+
+-- new_reseed_value as expression is not allowed
+DBCC CHECKIDENT(babel_3201_t2, RESEED, 4+5);
+GO
+
+-- new_reseed_value is out of tinyint datatype range
+DBCC CHECKIDENT(babel_3201_t_tinyint, RESEED, 256);
+GO
+
+DBCC CHECKIDENT(babel_3201_t_tinyint, RESEED, -1);
+GO
+
+-- new_reseed_value is out of smallint datatype range
+DBCC CHECKIDENT(babel_3201_t_smallint, RESEED, 32768);
+GO
+
+DBCC CHECKIDENT(babel_3201_t_smallint, RESEED, -32769);
+GO
+
+
+-- new_reseed_value is out of int datatype range
+DBCC CHECKIDENT(babel_3201_t_int, RESEED, 2147483648);
+GO
+
+DBCC CHECKIDENT(babel_3201_t_int, RESEED, -2147483649);
+GO
+
+-- new_reseed_value is out of bigint datatype range
+DBCC CHECKIDENT(babel_3201_t_bigint, RESEED, 9223372036854775808);
+GO
+
+DBCC CHECKIDENT(babel_3201_t_bigint, RESEED, -9223372036854775809);
+GO
+
+-- numeric/decimal datatypes are internally converted to bigint, so allowed
+-- range is same as that of bigint
+DBCC CHECKIDENT(babel_3201_t_numeric, RESEED, 9223372036854775808);
+GO
+
+-- When new_reseed_value is a float value, only value before the decimal is used
+DBCC CHECKIDENT(babel_3201_t_decimal, RESEED, 15.65);
+GO
+
+INSERT INTO babel_3201_t_decimal VALUES (8);
+GO
+
+SELECT * FROM babel_3201_t_decimal;
+GO
+
+begin tran;
+DBCC CHECKIDENT(babel_3201_t_decimal, RESEED, 10);
+INSERT INTO babel_3201_t_decimal VALUES (9);
+commit;
+go
+
+SELECT * FROM babel_3201_t_decimal;
+GO
+
+BEGIN TRAN;
+    DBCC CHECKIDENT(babel_3201_t_decimal, RESEED, 133ac);
+    INSERT INTO babel_3201_t_decimal VALUES (10);
+COMMIT;
+GO
+
+SELECT * FROM babel_3201_t_decimal;
+GO
+
+BEGIN TRAN;
+    DBCC CHECKIDENT(babel_3201_t_decimal, RESEED, 20);
+    INSERT INTO babel_3201_t_decimal VALUES (11);
+ROLLBACK;
+GO
+
+SELECT * FROM babel_3201_t_decimal;
```

Comment:
For ROLLBACK, we need to check what's the current identity value? In PG, it'll not be rolled-back - different from SQL Server.

Reply:
Added a `checkident(<table>, noreseed)` after transection rollback to check the current identity values, which is not rolled back.